.github/encrypt_secret.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# github actions doc: https://docs.github.com/en/actions/reference/encrypted-secrets
+
+# Prepare:
+#   install gpg
+#
+#   macos: brew install gpg
+#   ubuntu/debian: apt-get install -y gpg
+#   centos/RedHat: apt-get install -y gpg
+#
+# Usage:
+#
+# ./.github/envcrypt_secret.sh [development|production]
+
+set -e;
+
+ENV=$1
+if [ -f "config/.env.${ENV}.local" ]
+then
+    echo "encrypt config/.env.${ENV}.local to base64 format"
+    gpg --symmetric  --output tmp --cipher-algo AES256 config/.env.${ENV}.local
+    cat tmp | base64 -w 0 > config/.env.${ENV}.local.gpg
+    rm tmp
+    echo "encrypt complete, add .env.${ENV}.local.gpg to github secret >>GPG_SECRET_FILE<< and PASSWORD to github >>GPG_SECRET_PASSWORD<< for flat-web docker build"
```

Comment:
```suggestion
    echo "encrypt complete, add .env.${ENV}.local.gpg to github secret >>GPG_SECRET_FILE<< and PASSWORD to github >>GPG_SECRET_PASSWORD<< for flat docker build"
```

We shouldn't limit this script to `flat-web`, I'd prefer it to be a generic script